packages/idempotency/tests/helpers/populateEnvironmentVariables.ts

```diff
@@ -1,2 +1,7 @@
 // Reserved variables
-process.env.AWS_REGION = 'us-east-1';
+process.env._X_AMZN_TRACE_ID = '1-abcdef12-3456abcdef123456abcdef12';
+process.env.AWS_LAMBDA_FUNCTION_NAME = 'my-lambda-function';
+process.env.AWS_EXECUTION_ENV = 'nodejs16.x';
+process.env.AWS_LAMBDA_FUNCTION_MEMORY_SIZE = '128';
+process.env.AWS_REGION = 'eu-west-1';
+process.env._HANDLER = 'index.handler';
```

Comment:
(Question) Do we really need all of these env vars?  Wouldn't only `AWS_LAMBDA_FUNCTION_NAME` be enough?